src/kakarot/stack.cairo

```diff
@@ -4,68 +4,82 @@
 
 // Starkware dependencies
 from starkware.cairo.common.alloc import alloc
-from starkware.cairo.common.math import assert_lt_felt
+from starkware.cairo.common.math import assert_le
 from starkware.cairo.common.uint256 import Uint256
 from starkware.cairo.common.memcpy import memcpy
 
 // Internal dependencies
 from kakarot.constants import Constants
 from kakarot.model import model
 
+// New Stack dependencies
```

Comment:
why splitting "new" and "other"?

Reply:
It was for my own organization, just forgot to change